component/src/main/java/org/wso2/extension/siddhi/map/json/sourcemapper/JsonSourceMapper.java

```diff
@@ -65,35 +65,36 @@
+                + "to convert an incoming JSON message into a Siddhi event. Users can either send a pre-defined JSON "
+                + "format, where event conversion happens without any configurations, or use the JSON path to map" +
+                " from a custom JSON message.\n"
+                + "In default mapping, the JSON string of the event can be enclosed by the element \"event\" , " +
+                "though optional.",
         parameters = {
                 @Parameter(name = "enclosing.element",
                         description =
-                                "Used to specify the enclosing element in case of sending multiple events in same "
-                                        + "JSON message. \nWSO2 DAS will treat the child element of given enclosing "
-                                        + "element as events"
-                                        + " and execute json path expressions on child elements. \nIf enclosing" +
-                                        ".element "
-                                        + "is not provided "
-                                        + "multiple event scenario is disregarded and json path will be evaluated "
-                                        + "with respect to root element.",
+                                "This is used to specify the enclosing element when sending multiple " +
+                                        "events in the same "
+                                        + "JSON message. \nWSO2 DAS treats the child elements of a given enclosing "
+                                        + "element as events."
+                                        + " and executes the JSON path expressions on these child elements." +
+                                        " \nIf the enclosing.element is not provided then the multiple-event " +
+                                        "scenario is disregarded and the JSON path is evaluated based on " +
+                                        "the root element.",
                         type = {DataType.STRING},
                         optional = true,
                         defaultValue = "$"),
                 @Parameter(name = "fail.on.missing.attribute",
-                        description = "This can either have value true or false. By default it will be true. \nThis "
-                                + "attribute allows user to handle unknown attributes.\n By default if an json "
+                        description = "\nThis parameter allows users to handle unknown attributes.The value of this " +
+                                "can either be true or false. By default it is true. \n If a JSON "
                                 + "execution "
-                                + "fails or returns null DAS will drop that message. \nHowever setting this property"
+                                + "fails or returns null, DAS drops that message. \nHowever, setting this property"
```

Comment:
Remove the term DAS as this will no longer compatible with DAS. Use Siddhi or json mapper appropriately, as this can be used independently without WSO2 SP also.